### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by arunbalajeev** [ChiWeiHsiao/SphereNet-pytorch#2]
Thanks for your work!!
Could you please put up the pretrained weights for the spherenet model?

**Comment by ChiWeiHsiao** [comment]
Hi, you can download trained weight [here](https://drive.google.com/file/d/1z_QbT8vU4XLX5Jx8SC7fxbovuGD2Dlqs/view?usp=sharing).

**Comment by YamamotoSayaka** [comment]
Hi, @arunbalajeev , can you reach a similar performance?
I can only get 91% (9062/10000) accuracy with the pretrained model above.
Even if I trained from scratch, the result isn't good enough. 
any ideas? @ChiWeiHsiao

I'm using python3 and pytorch 0.4 .1

**Comment by tupoihybbjfiodkxmhcid123456** [comment]
The pre-trained model no longer exists, could you share it again? Thank you very much.
